CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+How to contribute
+=================
+
+Thanks for your interest in contributing to Kinto!
+
+## Reporting Bugs
+
+Report bugs at https://github.com/Kinto/kinto-http.py/issues/new
+
+If you are reporting a bug, please include:
+
+ - Any details about your local setup that might be helpful in troubleshooting.
+ - Detailed steps to reproduce the bug.
+
+
+## Ready to contribute?
+
+### System Requirements
+
+Depending on the platform and chosen configuration, some libraries or extra services are required.
+
+The following commands will install necessary tools for cryptography and Python packaging like Virtualenv.
+
+#### Linux
+On Debian / Ubuntu based systems:
+
+```bash
+apt-get install libffi-dev libssl-dev python-dev python-virtualenv
+```
+
+#### OS X
+Assuming [brew](http://brew.sh/) is installed:
+
+```bash
+brew install libffi openssl pkg-config python
+
+pip install virtualenv
+```
+
+
+### Getting Started
+
+ -  Fork the repo on GitHub.
```

Comment:
I think it is best to clone from the Kinto/kinto-http.py repository and then add your remote:

```
git clone git@github.com:Kinto/kinto-http.py.git
git remote add {your_name} git@github.com:{your_name}/kinto-http.py.git
```

Then you can:

```
git checkout master
git pull origin master
git checkout -b issue_number-bug-title
git push -u {your_name} issue_number-bug-title
```